rholang/src/main/scala/coop/rchain/rholang/interpreter/compiler/normalizer/processes/PBundleNormalizer.scala

```diff
@@ -28,13 +28,20 @@ object PBundleNormalizer {
     def error(targetResult: ProcVisitOutputs): F[ProcVisitOutputs] = {
       val errMsg = {
         def at(variable: String, sourcePosition: SourcePosition): String =
-          variable + " line: " + sourcePosition.row + ", column: " + sourcePosition.column
+          variable + " at " + sourcePosition.row + ":" + sourcePosition.column
```

Comment:
Definition of SourcePosition already defines `toString` method which will be called when instance is part of string interpolation.

https://github.com/rchain/rchain/blob/c7b0414ff3ee4249099deec39810e0a835d67336/rholang/src/main/scala/coop/rchain/rholang/interpreter/compiler/SourcePosition.scala#L3-L5

String interpolation should be used to define this function also.
https://docs.scala-lang.org/overviews/core/string-interpolation.html